packages/components/datepicker/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { getStencilJestConfig } from '@ovhcloud/ods-common-testing';
+
+const config = getStencilJestConfig({
+  args: process.argv.slice(2),
+});
+
+config.moduleNameMapper = {
```

Comment:
do not update the config directly, you have an `options` attr that you can use on getStencilJestConfig